include/nlohmann/detail/conversions/to_json.hpp

```diff
@@ -290,31 +290,31 @@ void to_json(BasicJsonType& j, typename BasicJsonType::object_t&& obj)
 template <
     typename BasicJsonType, typename T, std::size_t N,
     enable_if_t<not std::is_constructible<typename BasicJsonType::string_t,
-                const T (&)[N]>::value,
+                const T(&)[N]>::value,
```

Comment:
Can you make sure you ran `make pretty` - the indentation and whitespace looks odd in the whole PR. This is not a blocker, just a nice-to-have.

Reply:
Sorry.
I was previously using MSYS 32-Bit which didn't have astyle installed so amalgamate worked without issues.
Switching to MSYS 64 Bit solves this issue.

Reply:
The diff still looks weird, but that's OK for now.